### PR TITLE
Add maintenance plan

### LINKED ISSUE #1223: Create & Document our Long Term Maintenance Plan
* **What are we trying to do?**

Come up with our plans for what the long term maintenance mode looks like, and document it.

* **What feature or behavior is this required for?**

Long term health of the mature app.  
Being there for our awesome funds.
@colinxfleming getting more sleep!  

* **How could we solve this issue? (Not knowing is okay!)**

A solid doc with the decisions we make in this thread.  
Adding it to the readme

* **Anything else?**

Some starter thoughts:

  - how can new funds get on board? Has this changed?
  - how can existing funds request changes? Security vs important vs nice to have?
  - who has the patch pumpkin? Does it rotate?
  - how do we accept new features? Only fully tested PRs?
  - def more things

@colinxfleming @DarthHater @lomky @mebates @lwaldsc (others I'm forgetting please loop in)

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Document and formalize our discussion from the summer about maintenance and new development going forward.

I addressed the followup items as well - I think limited app controls could be challenging if we did it across the board but we can and should give app management to anyone who would like it. Also added restart instructions.

This pull request makes the following changes:
* Adds maintenance roadmap doc
* Adds FAQ doc with app reset instructions

It relates to the following issue #s: 
* Fixes #1223 

@NerdyGirl537 @lwaldsc @alisonnjones @lomky can you all review this and flag any changes or omissions?

### CODE REVIEW

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Our maintenance plan
+
+This roadmap outlines responsibilities for administration of funds in the DCAF-administered Heroku pipeline (e.g. where DCAF administers DARIA for them), and roadmapping new development.
+
+## Administration
+
+* The engineering team has set up a series of monitors for applications, and is responsible for keeping an eye on them and troubleshooting when something goes wrong.
```
Comment (lomky) [inline docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Which applications? All deployed applications, specific Funds applications?

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* The engineering team is responsible for applying security fixes and other regular maintenance.
+* The DCAF case management team is responsible for communicating support requests to the engineers in the slack channel.
```
Comment (lomky) [inline docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Link the slack channel?

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* We can allow fund stakeholders access to management of the application so they can restart it if they like; funds should reach out to the engineering team or DARIA project lead if they're interested.
+
+## Developing new features
+
+* Every once in awhile (probably six to nine months), the DARIA project lead and DCAF data board member will poll DCAF and other member funds and see what development priorities or pain points have popped up, and select one or two priorities to work on.
+* When this happens, the DARIA project lead is responsible for assembling engineers and pushing it through.
+* A core team of engineers will continue to review pull requests.
+
+## Getting new funds on DARIA
```
Comment (lomky) [inline docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Do we have another writeup on this we can link to? If not, can we explain that it's relatively painless?

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 ## Next major project milestone: Sharing with other funds
 
-Major new feature development on this project has more or less drawn to a close. If you are an abortion fund interested in whether this would work for you, hit us up and let us know!
+Major new feature development on this project has more or less drawn to a close. Our general administration/maintenance plan is [here](docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md). If you are an abortion fund interested in whether this would work for you, hit us up and let us know!
 
 ## [Come hang out with us on slack!](https://codefordc.slack.com/messages/dcaf_case_management)
```
Comment (lomky) [inline README.md]: There's a line further down that say's we're a regular presence at CodeForDC meetups. Maybe we should redact that on this pull?

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md (hunk 3 above)
Comment (lomky) [reply docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Found it: https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SETTING_UP_A_NEW_INSTANCE.md#by-the-way

File: README.md (hunk 4 above)
Comment (lwaldsc) [reply README.md]: Probably just easier to link to https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md (hunk 1 above)
Comment (lwaldsc) [reply docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Probably just easier to link to https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md

**Comment by lwaldsc** [comment]
Overall, I think this works well. Kat caught some great things.

File: README.md (hunk 4 above)
Comment (colinxfleming) [reply README.md]: great catch! Fixed in b199aaec

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md (hunk 1 above)
Comment (colinxfleming) [reply docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: I think 677822e5 addresses @lomky 's point here!

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md (hunk 2 above)
Comment (colinxfleming) [reply docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Good call! Fixed in f3508633

File: docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md (hunk 3 above)
Comment (colinxfleming) [reply docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md]: Added a link and an additional bullet in 70d5f2f4 . This is a great call.

**Comment by colinxfleming** [comment]
Made suggested changes per @lomky 's great comments. 

@NerdyGirl537 @alisonnjones I'd love to smash merge on this by Friday so we can ~ make it official ~, would appreciate eyes before then if you can!

**Comment by alisonnjones** [comment]
The plan looks awesome! I fully support the work outlined in it and the responsibilities breakdown. It's not super detailed, but I think it provides us with all we need at this point. Love it.

File: README.md (hunk 4 above)
Comment (NerdyGirl537) [reply README.md]: On line 16, the Join via the Code for DC website link gets a 404 error

**Comment by NerdyGirl537** [comment]
In the Code of Conduct, we list Colin and Mollie as project maintainers and Colin as the main point of contact. Do we want to continue with that? If not, who do we need to designate as project maintainers?

**Comment by colinxfleming** [comment]
@NerdyGirl537 I'm fine with our being listed as project maintainers for the time being, with respect to the code of conduct, though we should definitely be funneling basically everything else through you. Great flag!

**Comment by colinxfleming** [comment]
Smashing merge on this. Thanks all!

**Pull request merged**